src/static.ts

```diff
@@ -431,6 +431,8 @@ export interface ConfigOptions {
   }>;
   tabBarButtons?: TabBarMainAction[];
   maxTabs: number;
+  maxReachedOverlayDuration?: number;
```

Comment:
what do you think about naming this as `maxTabsTooltipDuration`  I feel like the variable should contain `tabs` in the name to make it easier to reason with

Reply:
Sure, I've stick to the previous naming, but sounds better we tie to the tooltip. I'll include in the next rev.